gradle/libs.versions.toml

```diff
@@ -41,7 +41,7 @@ kotlinx-coroutines-test = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-t
 
 kotlinx-binaryCompatibilityValidator = "org.jetbrains.kotlinx:binary-compatibility-validator:0.13.2"
 
-ktlint = "com.pinterest:ktlint:0.50.0"
+ktlint = "com.pinterest.ktlint:ktlint:1.0.1"
```

Comment:
This should be `"com.pinterest.ktlint:ktlint-cli:1.0.1"` for now, see https://github.com/pinterest/ktlint/releases/tag/1.0.0

Reply:
Want to send a PR?

Reply:
It's #325.